test/clj/cider/nrepl/middleware/refresh_test.clj

```diff
@@ -61,14 +65,22 @@
       (is (:error response)))))
 
 (deftest after-arg
```

Comment:
Maybe we should use `after-arg-fn` as our term in the tests. "After arg" sounds a bit odd.